Home.md

```diff
@@ -44,11 +55,11 @@ Just go to [http://demo.dswarm.org](http://demo.dswarm.org) and try it out or se
 ### Installation and Configuration
 
 It might be a good idea to run d:swarm locally to get full insight into the processes of our application.
-Installation instructions can be found in the [[Server Install]] or [[Developer Install]] guide, [[d:swarm Configuration|dswarm Configuration]] provides details on how to configure the system.
+Installation instructions can be found in the [[Developer Install]] guide, [[d:swarm Configuration|dswarm Configuration]] provides details on how to configure the system. (For productive use of d:swarm see [[Server Install]].)

```

Comment:
I would put the Server Install reference first and the the Developer Install afterwards, because we first intend institutions to deploy d:swarm and the we would like to attract developers to contribute to it